### PR TITLE
Sort DataFrame columns from motif finding

### PR DESCRIPTION
**Pull request by felixcheung**
Had another attempt to reparse pattern for name, but with the need to deduplicate for pattern like `(a)-[]->(b); (b)-[]->(c); (c)-[]->(a)` which is already implemented in `seen()` thought it might be better to have a single pass at the cost of slightly less readable code with pair as return values.

Thought? #45

### CODE REVIEW

File: src/main/scala/org/graphframes/GraphFrame.scala [hunk 1]
```diff
@@ -367,12 +369,13 @@ class GraphFrame private(
   private def findSimple(
       prevPatterns: Seq[Pattern],
       prevDF: Option[DataFrame],
-      remainingPatterns: Seq[Pattern]): DataFrame = {
+      prevNames: Seq[String],
+      remainingPatterns: Seq[Pattern]): (DataFrame, Seq[String]) = {
```
Comment (jkbradley) [inline src/main/scala/org/graphframes/GraphFrame.scala]: Document new argument & return value

**Comment by jkbradley** [comment]
That's really the only issue I see. Otherwise, looks good.  Thanks!

**Comment by felixcheung** [comment]
done, thanks!

**Comment by jkbradley** [comment]
LGTM
Thanks for the fix!

**Pull request merged**